### PR TITLE
Fix resource less method

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
Fix: #1036

**Comment by k82cn** [comment]
Can you help to add some test case for the differentiation?

**Comment by hzxuzhonghu** [comment]
@k82cn This is ready to go

File: pkg/scheduler/api/resource_info.go [hunk 1]
```diff
@@ -224,40 +224,46 @@ func (r *Resource) Multi(ratio float64) *Resource {
...
+				isLess = true
+			}
 		}
 	}
 
-	return true
+	return isLess
 }
 
 // LessEqualStrict checks whether a resource is less or equal than other
```
Comment (shinytang6) [inline pkg/scheduler/api/resource_info.go:269]: l think for the issue #1036, maybe function `LessEqualStrict ` and `LessEqual ` can be changed together)

**Comment by zen-xu** [comment]
This pr still has bug

```go
func main() {
	allocated := &api.Resource{
		MilliCPU:        1000,
		Memory:          2000,
	}

	deserved := &api.Resource{
		MilliCPU:        2000,
		Memory:          3000,
		ScalarResources: map[v1.ResourceName]float64{
			"nvidia.com/gpu": 4500,
			"hugepages-1Gi": 0,
			"hugepages-2Mi": 0,
		},
	}
	fmt.Println(allocated.Less(deserved))  // false, should be true
}
```

**Comment by k82cn** [comment]
please use Rat (xref: https://golang.org/src/math/big/rat.go ) for that.

**Pull request closed without merging**